html-forms/index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title> Html Forms</title>
+  <body>
+    <h1> Coding Form!
+    </h1>
+    <div>
+      <label for="email-address">Email</label> <br>
+      <input type="text" name="email" size="15" maxlength="50" />
+    </div><br>
```

Comment:
Hi Evan! Please only use br's when trying to separate paragraphs onto new lines of text. Use divs to move form elements onto new lines